lib/views/feedbacks_slack_view.ex

```diff
@@ -0,0 +1,8 @@
+defmodule Aprb.Views.FeedbacksSlackView do
+
+  def render(event) do
+    %{
+      text: "#{event["subject"]["display"]} #{event["verb"]} #{event["properties"]["message"]} from #{event["properties"]["url"]}",
```

Comment:
Careful connecting a local consumer to staging RabbitMQ, as I think there have been problems disconnecting these later on when a real staging consumer wants to step in.

@ashkan18 did I remember this correctly? What's the recommended practice around different environments and their bindings?